int32 num

time stamp

float64[] x_min

float64[] x_max

float64[] y_min

float64[] y_max
